pkg/agent/accesscontrollisthandler.go

```diff
@@ -96,6 +99,23 @@ func (j *aclUpdateHandlerJob) statusUpdate() {
 	}
 }
 
+func isItemInArray(searchArray []string, item string) bool {
+	if len(searchArray) == 0 {
+		return false
+	}
+	if len(searchArray) == 1 {
+		return searchArray[0] == item
+	}
+	midPoint := len(searchArray) / 2
+	if item == searchArray[midPoint] {
+		return true
+	}
+	if item < searchArray[midPoint] {
+		return isItemInArray(searchArray[:midPoint], item)
+	}
+	return isItemInArray(searchArray[midPoint:], item)
+}
```

Comment:
Liked the O(log n) complexity :)

Reply:
I was running that sort func for a reason

Reply:
moving this func to util